src/main/java/com/umbr3114/controllers/DropController.java

```diff
@@ -56,8 +63,33 @@
         dropCollection = new CollectionFactory<DropModel>(ServiceLocator.getService().dbService(), DropModel.class).getCollection();
         dropCollection.save(drop);
 
-        return new GeneralResponse(HttpStatus.OK_200, "successful");
+        return "okay";
+    });
+
+    public static Route updateDrop = ((request, response) -> {
+        String dropId;
+        String description = null;
+        RequestParamHelper helper = new RequestParamHelper(request);
+        JacksonMongoCollection<DropModel> update;
+        dropId = helper.valueOf("dropId");
+        update = new CollectionFactory<DropModel>(ServiceLocator.getService().dbService(), DropModel.class).getCollection();
+        update.updateOne(eq("_id", new ObjectId("dropId")), combine (set("title", 1), set("topic", ""), currentDate("lastModified")));
+        if (description == null || description.isEmpty());
+        halt(HttpStatus.NOT_FOUND_404, "did not update");
+        return "something";
+
```

Comment:
This route doesn't even work. I don't think it was even tested..

Line 76: the ObjectId's don't even take in a variable, they are taking in a "string" and causing the route to throw an exception. Unfortunately, this doesn't even allow the updateOne function to even be evaluated to see if it works.

dropId is set using input from the request, but is never used. 

Lastly.. This route isn't updating anything. The Intent of this route was that so a drop owner could modify the topic of this Route, but it doesn't accept input that resembles a topic.

This route is going to have to be re-thought